lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -1393,4 +1399,21 @@ public function __clone()
             $this->params[$name] = clone $param;
         }
     }
+
+    /**
+     * Filters out empty values from the given argument list.
+     * Fixes a BC break introduces in 2.11: https://github.com/doctrine/dbal/issues/4282
+     *
+     * @param array<mixed> $args
+     *
+     * @return array<mixed>
+     *
+     * @todo remove for 3.0
+     */
+    private static function filterEmptyArgs(array $args): array
```

Comment:
Why does the method need to be static?

Reply:
Because phpcs wants it so!

Reply:
Please double-check. I can't reproduce this by making the method non-static.

Reply:
It's [here](https://github.com/BenMorel/dbal/runs/1151940985):
> [error]Closure not using "$this" should be declared static.